.travis.yml

```diff
@@ -68,3 +80,11 @@ script:
     - go test -c -tags "apparmor seccomp `./btrfs_tag.sh` `./libdm_tag.sh` `./ostree_tag.sh` `./selinux_tag.sh`" ./cmd/buildah
     - tmp=`mktemp -d`; mkdir $tmp/root $tmp/runroot; sudo PATH="$PATH" ./buildah.test -test.v -root $tmp/root -runroot $tmp/runroot -storage-driver vfs -signature-policy `pwd`/tests/policy.json -registries-conf `pwd`/tests/registries.conf
     - cd tests; travis_wait 60 sudo PATH="$PATH" ./test_runner.sh
```

Comment:
@TomSweeneyRedHat @nalind is it ok to move this and the docker tests above inside the Fedora container as well?

Reply:
I'd rather keep the docker tests in the top level container that travis spins up.  The registry that it sets up is then used later by buildah authentication tests in the ./buildah/tests directory.

Reply:
ack on docker tests, but how about moving the buildah build and test stuff (the lines right below the docker operations) to the fedora container? just that I find it a little annoying that those work well on upstream Travis build, but fail on my fork. I think those could work consistently within Fedora (but need to verify it though)

Reply:
My only hesitancy with that move is the work I'm trying to pull off with the codecov.io.  If I ever manage to get that to work, the codecoverage file needs to be saved to a place that travis can get to it and then copy to codecov.io.   That's a lot easier without the tests running in a container, but is not insurmountable.  But then again, I'm not getting it to work for Buildah even with out a container at the moment.

So long story hopefully shortened, if it makes life easier for you to have them in a container rather than the top level travis, then go ahead and move them.